docs/core/dialogs/dialog.md

```diff
@@ -89,9 +89,12 @@ function openDialog() {
         title: 'Dialog title',
         contentComponent: ExampleDialogComponent,
         componentData: { nodeId: 'nodeId', name: 'node name' } // any data can be passed
+        dataOnConfirm$: of({ nodeId, data: {} })
```

Comment:
Okay so what is the use case for this change?

Reply:
if you click confirm button and need to pass any data to after close action